test/src/unit-testsuites.cpp

```diff
@@ -384,6 +384,46 @@ TEST_CASE("json.org examples")
         json j;
         CHECK_NOTHROW(f >> j);
     }
+    SECTION("FILE 1.json")
+    {
+        auto f = fopen("test/data/json.org/1.json", "r");
```

Comment:
This is a minor issue, but will avoid having to write "fclose" at the bottom, and technically also makes sure that things get cleaned up even if an exception gets thrown. (Not a serious issue for test code, but it's still nice to have unit tests always clean up their resources in all cases).

```
std::unique_ptr<FILE, decltype(&fclose)> f(fopen("test/data/json.org/1.json", "r"), &fclose);
```
You'll have to do ```j.parse(f.get())``` in the lines below.

There are ways to make this prettier if we want, but this is bare minimum I would recommend.

Reply:
updated as per your comment